spec/errorSpec.js

```diff
@@ -1,6 +1,5 @@
 var webdriver = require('selenium-webdriver');
 var common = require('./common.js');
-require('../index.js');
 
```

Comment:
Nope, it's taken care of in common.js. Though I can move it out of there if that's more clear.